### DISCUSSION TITLE
Setting Homebrew PATH statements after install breaks links to all other cli apps

### DISCUSSION THREAD

**Discussion by dburnett** [Homebrew/discussions#3926] [Getting started]
I have Homebrew installed on OS X Monteray (12.6.1).  The PATH statements the brew install recommends at the end of the install breaks my links to all other apps in ZSH.  When I add these statements: "echo '#Set PATH, MANPATH, etc., for Homebrew.' >> /Users/<home-dir>/ .xprofile' and echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/home-dir/.zprofile" I cannot run others commands such as 'cat', 'ls', or 'cd'. 

How can I add Homebrew to my current PATH statement without breaking everything else?

**Comment by carlocab** [comment]
This seems to be specific to your configuration, since many other users do the same thing without the same issue. This means that only you can work out how to fix this, I'm afraid.
